.yarnrc.yml

```diff
@@ -1,7 +1,8 @@
 nodeLinker: node-modules
 
 plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-outdated.cjs
-    spec: "https://mskelton.dev/yarn-outdated/v3"
+  - checksum: 53df621960dbcb9cc3953b45376e0d7e0fbcb870eec71ebb8ffa575c2e834349fdcfa255c46b8fa3fff2c65f26e55c7339c2cb57ee80f92d62c9e181bcb7792d
+    path: .yarn/plugins/@yarnpkg/plugin-outdated.cjs
+    spec: "https://mskelton.dev/yarn-outdated/v4"
```

Comment:
@avo When you update projects at Widen to Yarn 4, the Yarn outdated plugin now supports Yarn v4. The v3 version of the plugin seems to also work, but just an FYI.